Redis - SearchCommandsTest.testAggregation is flaky

`io.quarkus.redis.datasource.SearchCommandsTest.testAggregation` is flaky and sometimes get four results instead of three, making the test fail at this step: https://github.com/quarkusio/quarkus/blob/f76d3e30eef1dbc0796bb09571b5564c3dbb6949/extensions/redis-client/runtime/src/test/java/io/quarkus/redis/datasource/SearchCommandsTest.java#L452-L464 .

```
2022-12-07T01:15:25.1412106Z [ERROR] io.quarkus.redis.datasource.SearchCommandsTest.testAggregation  Time elapsed: 10.564 s  <<< ERROR!
2022-12-07T01:15:25.1412782Z org.awaitility.core.ConditionTimeoutException: 
2022-12-07T01:15:25.1414120Z Assertion condition defined as a lambda expression in io.quarkus.redis.datasource.SearchCommandsTest 
2022-12-07T01:15:25.1414590Z expected: 3
2022-12-07T01:15:25.1630037Z  but was: 4 within 10 seconds.
2022-12-07T01:15:25.1631410Z 	at org.awaitility.core.ConditionAwaiter.await(ConditionAwaiter.java:167)
2022-12-07T01:15:25.1631881Z 	at org.awaitility.core.AssertionCondition.await(AssertionCondition.java:119)
2022-12-07T01:15:25.1632324Z 	at org.awaitility.core.AssertionCondition.await(AssertionCondition.java:31)
2022-12-07T01:15:25.1632748Z 	at org.awaitility.core.ConditionFactory.until(ConditionFactory.java:985)
2022-12-07T01:15:25.1633283Z 	at org.awaitility.core.ConditionFactory.untilAsserted(ConditionFactory.java:769)
2022-12-07T01:15:25.1633800Z 	at io.quarkus.redis.datasource.SearchCommandsTest.testAggregation(SearchCommandsTest.java:452)
```

--- a/extensions/redis-client/runtime/src/test/java/io/quarkus/redis/datasource/SearchCommandsTest.java
+++ b/extensions/redis-client/runtime/src/test/java/io/quarkus/redis/datasource/SearchCommandsTest.java
@@ -455,7 +455,7 @@ public class SearchCommandsTest extends DatasourceTestBase {
                     .groupBy(new AggregateArgs.GroupBy().addProperty("@day").addProperty("@country").addReduceFunction("count",
                             "num_visits"))
                     .sortBy(new AggregateArgs.SortBy().ascending("@day").descending("@country")));
-            assertThat(result.count()).isEqualTo(3);
+            assertThat(result.count()).isGreaterThanOrEqualTo(3);
             assertThat(result.documents()).allSatisfy(d -> {
                 assertThat(d.property("day").asInteger()).isPositive();
                 assertThat(d.property("country").asString()).isNotNull();

